packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1026,8 +1029,6 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         return {
```

Comment:
Consider making it a variable that is initialized in constructor - it seems wasteful to use getter that continuously creates same object over and over again.